README.md

```diff
@@ -1001,3 +1001,34 @@ distributions:
   - `sign-key-name: str` --- name of the signing key to use. For test keys this becomes the subject of the self-signed certificate.
   - `test-sign: bool` --- code signing type, `true` (default) or `false`. Test signing generates ephemeral self-signed
     keys for each component. Production signing uses an already existing key signed by a public CA (TODO: HSM).
+
+### Cross-distribution dependencies
+
+To perform cross-compilation between distributions, you must declare dependencies using the `needs: List[Dict]` structure within the appropriate distribution stage.
```

Comment:
Cross-compilation is the opposite of what you do here. We specifically do _not_ use cross-compilation anymore (we used to build Windows tools on Linux, but not anymore).